From f8181dbd6103d8081c48d593a2e1cb10a1b690d7 Mon Sep 17 00:00:00 2001
From: Tanya Finkel <tfinkel@codeaurora.org>
Date: Thu, 13 Feb 2014 16:00:46 +0200
Subject: [PATCH 1/3] Audio: insert support for SBC8084 platform

part of the audio device on the SBC8084 are various from other platforms

Change-Id: Ic00eb2d5be2986c5dbcc175a072e96186654a855
---
 mixer_paths.xml |   18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/mixer_paths.xml b/mixer_paths.xml
index 9301a42..c606c65 100644
--- a/mixer_paths.xml
+++ b/mixer_paths.xml
@@ -726,6 +726,14 @@
         <ctl name="IIR1 INP1 MUX" value="DEC5" />
     </path>
 
+    <path name="adc5">
+        <ctl name="AIF1_CAP Mixer SLIM TX7" value="1"/>
+        <ctl name="SLIM_0_TX Channels" value="One" />
+        <ctl name="SLIM TX7 MUX" value="DEC2" />
+        <ctl name="DEC2 MUX" value="ADC5" />
+        <ctl name="IIR1 INP1 MUX" value="DEC2" />
+    </path>
+
     <path name="dmic3">
         <ctl name="AIF1_CAP Mixer SLIM TX7" value="1"/>
         <ctl name="SLIM_0_TX Channels" value="One" />
@@ -791,6 +799,11 @@
         <ctl name="DEC9 Volume" value="111" />
     </path>
 
+    <path name="speaker-mic-sbc">
+        <path name="adc5" />
+        <ctl name="ADC5 Volume" value="20" />
+    </path>
+
     <path name="handset">
         <ctl name="SLIM RX1 MUX" value="AIF1_PB" />
         <ctl name="SLIM RX6 MUX" value="AIF2_PB" />
@@ -812,6 +825,11 @@
         <ctl name="ADC1 Volume" value="20" />
     </path>
 
+    <path name="handset-mic-sbc">
+        <path name="adc5" />
+        <ctl name="ADC5 Volume" value="20" />
+    </path>
+
     <path name="anc-handset">
         <ctl name="ANC Function" value="ON" />
         <ctl name="SLIM RX1 MUX" value="AIF1_PB" />
-- 
1.7.9.5

